cmd/kube-scheduler/app/options/options.go

```diff
@@ -236,6 +236,9 @@ func (o *Options) Config() (*schedulerappconfig.Config, error) {
 	}
 
 	coreBroadcaster := record.NewBroadcaster()
+	if c.ComponentConfig.SchedulerName == "" {
```

Comment:
wait, can you revert this? I'm adding the checks in ApplyTo #84815

Reply:
Yup, removed, can you re-lgtm?